analysis_options.yaml

```diff
@@ -10,4 +10,4 @@ analyzer:
     - lib/**/*.gr.dart
     - lib/assets/**
     - tools/bricks/**/__brick__/**
-
+    - lib/l10n/**
```

Comment:
We want to exclude from analysis_options only generated file, not all folder. Also, we need to add generated files to .gitignore.